### ISSUE TITLE
Unable to run laravel-stats when Swoole Extention is instaleld

### ISSUE THREAD

**Issue by james-wong123** [stefanzweifel/laravel-stats#206]
"wnx/laravel-stats": "^2.9"
"laravel/sail": "^1.14",
"laravel/framework": "^9.0",

I am running on docker with laravel sail, after installing your package I run `sail php artisan stats`, and this returned

```
ErrorException 

  file_get_contents(@swoole-src/library/functions.php): Failed to open stream: No such file or directory

  at vendor/phploc/phploc/src/Analyser.php:161
    157▕         if ($countTests) {
    158▕             $this->preProcessFile($filename);
    159▕         }
    160▕
  ➜ 161▕         $buffer = file_get_contents($filename);
    162▕         $this->collector->incrementLines(substr_count($buffer, "\n"));
    163▕         $tokens    = token_get_all($buffer);
    164▕         $numTokens = count($tokens);
    165▕

      +26 vendor frames 
  27  artisan:37
      Illuminate\Foundation\Console\Kernel::handle()
```

**Comment by stefanzweifel** [comment]
This issue should have been fixed a while a go: https://github.com/stefanzweifel/laravel-stats/issues/195#issuecomment-830780195

Could you republish the config file or update it accordingly and test again.

**Comment by stefanzweifel** [comment]
Ah sorry, that PR just excludes the Namespace. You seem to be having issues reading a file.

Will investigate further.

**Comment by james-wong123** [comment]
@stefanzweife, no luck same thing happened 

> This issue should have been fixed a while a go: [#195 (comment)](https://github.com/stefanzweifel/laravel-stats/issues/195#issuecomment-830780195)
> Could you republish the config file or update it accordingly and test again.

**Comment by stefanzweifel** [comment]
Will try to reproduce this bug in the coming hours.

@james-wong123  Did you try adding `@swoole-src` to the [`stats.exclude`](https://github.com/stefanzweifel/laravel-stats/blob/main/config/stats.php#L17-L20) config?

**Comment by stefanzweifel** [comment]
I could reproduce the bug but the `stats.exclude` config doesn't seem to work here. 🤔

**Comment by stefanzweifel** [comment]
Thanks again for reporting. I've fixed the issue in #207.
A new release has just been tagged: https://github.com/stefanzweifel/laravel-stats/releases/tag/v2.9.1

Run `sail composer upgrade wnx/laravel-stats` and you should be good to go.
